tasks/db_deploy/install/orca_sql.py

```diff
@@ -623,4 +625,283 @@ def files_table_sql() -> TextClause:
         -- Grants
         GRANT SELECT, INSERT, UPDATE, DELETE ON files TO orca_app;
     """
+    )
+
+# ----------------------------------------------------------------------------
+# ORCA SQL used for creating ORCA internal reconciliation tables
+# ----------------------------------------------------------------------------
+
+def reconcile_status_table_sql() -> TextClause:
+    """
+    Full SQL for creating the reconcile_status table. 
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_status table.
+    """
+    return text(
+        """
+        -- Create table
+        CREATE TABLE IF NOT EXISTS reconcile_status  
+        ( 
+          id   	int2 NOT NULL
+        , value	text NOT NULL
+        , CONSTRAINT PK_reconcile_status PRIMARY KEY(id)
+        , CONSTRAINT UNIQUE_reconcile_status_value UNIQUE (value)
+        );
+
+        -- Comments
+        COMMENT ON TABLE reconcile_status 
+            IS 'Reference table for valid status values and status order.';
+        COMMENT ON COLUMN reconcile_status.id 
+            IS 'Status ID';
+        COMMENT ON COLUMN reconcile_status.value 
+            IS 'Human readable status value';
+
+        -- Upsert the data lookup rows for the table
+        INSERT INTO reconcile_status VALUES (1, 'getting S3 list')
+            ON CONFLICT (id) DO NOTHING;
+        INSERT INTO reconcile_status VALUES (2, 'staged')
+            ON CONFLICT (id) DO NOTHING;
+        INSERT INTO reconcile_status VALUES (3, 'generating reports')
+            ON CONFLICT (id) DO NOTHING;
+        INSERT INTO reconcile_status VALUES (4, 'error')
+            ON CONFLICT (id) DO NOTHING;
+        INSERT INTO reconcile_status VALUES (5, 'success')
+            ON CONFLICT (id) DO NOTHING;
+        """
+    )
+
+
+def reconcile_job_table_sql() -> TextClause:
+    """
+    Full SQL for creating the reconcile_job table. 
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_job table.
+    """
+    return text(
+        """
+        -- Create reconcile_job table
+        CREATE TABLE IF NOT EXISTS reconcile_job  
+        ( 
+          id                     	BIGSERIAL NOT NULL
+        , orca_archive_location  	text NOT NULL
+        , status_id              	int2 NOT NULL
+        , inventory_creation_time	timestamp with time zone NOT NULL
+        , start_time             	timestamp with time zone NOT NULL
+        , last_update            	timestamp with time zone NOT NULL
+        , end_time               	timestamp with time zone NULL
+        , error_message          	text NULL
+        , CONSTRAINT PK_reconcile_job PRIMARY KEY(id)
+        , CONSTRAINT FK_reconcile_job_status FOREIGN KEY(status_id) REFERENCES reconcile_status(id)
```

Comment:
Depending on what comes out of my PR, delete_marker and the keys may change.

Reply:
https://github.com/nasa/cumulus-orca/pull/241#discussion_r805021450
Remove constraints and create per partition.

Reply:
Updated table name definition and moved constraints to be created at the partition level in order to handle the older PostgreSQL engine (10.14) that PG Aurora Serverless uses.